src/main/scala/io/phdata/streamliner/configuration/parser/JDBCParser.scala

```diff
@@ -17,7 +17,7 @@ object JDBCParser {
     val catalog = SchemaCrawlerImpl.getCatalog(jdbc, password)
 
     val tables =
-      catalog.getSchemas.asScala.find(s => s.getFullName.equals(jdbc.schema)) match {
```

Comment:
I think I had an issue where a source possibly Oracle required or MSSQL needed the `getFullName` vs `getName` to match correctly.

Perhaps later versions of schema crawler fixed the issue I ran into with `getFullName` vs `getName`.

Reply:
Gotcha. The new version of SC prefixes the "full" name with the "type" so it's a more complex string.

Maybe we use "contains" as instead? This could pull in some schemas which aren't needed due to false positives but should work the vast majority of the time?

Reply:
@tfokken - it may have been SQL Server. It has strange naming, where the catalog is something like "AdventureWorks", and schema is "dbo". Then the full name would be "AdventureWorks.dbo", and name would be "dbo". Name would match pretty much anything else, for example, "Northwind.dbo".

Reply:
If that is what you are thinking of, then this is really not a SchemaCrawler thing, but a Microsoft SQL Server thing, and you should still use `s.getFullName`.

Reply:
I bet that is it. I think we'll use contains against fullname as I have in this PR: https://github.com/phdata/streamliner/pull/22